cephci/utils/configs.py

```diff
@@ -0,0 +1,99 @@
+import yaml
```

Comment:
also, can we have doc strings for these methods? Mentioning the params, possible values etc

Reply:
Done